tests/foreman/api/test_user.py

```diff
@@ -6,19 +6,19 @@
 """
 import ddt
 from fauxfactory import gen_string
+from nailgun import client
 from random import randint
 from requests.exceptions import HTTPError
-from robottelo import entities
+from robottelo.common.helpers import get_server_credentials
 from robottelo.common import decorators
+from robottelo import entities
```

Comment:
the root namespace should be before others: `from robottelo` should come before from `robottelo.something`.